devel/uv/Portfile

```diff
@@ -16,11 +16,19 @@ description             Extremely fast Python package and project manager
 long_description        {*}${description}, written in Rust.
 
 checksums               ${distname}${extract.suffix} \
-                        rmd160  ac1713733193bc4a46e78e3ebc3cd86a98fd0e44 \
-                        sha256  493bd38750b397ed5c361fdb28a007186d13e5fa7d46bc997b3a1eff12a839f5 \
-                        size    3139617
+                        rmd160  4d45f726f57b785c2d9a9f882507fdfa18dba0ed \
+                        sha256  4f581e0903285e3228f55967f7094f3311a06aa3ec4e4ea3bf7438702fb776d6 \
+                        size    3145749
 
+# Fix opportunistic linking with libiconv
+#
+# Ideally, the rust PG should handle any opportunistic linking with
+# MacPorts' libiconv, but that's a ticket that someone else should
+# open. Almost any Rust port links with libiconv, so it should be a
+# depends_lib, not a depends_build.
 depends_build-append    path:bin/pkg-config:pkgconfig
+depends_lib-append      path:lib/liblzma.dylib:xz \
```

Comment:
is there another port that would provide the dylib? If not, please just use the normal port dependency declaration

Reply:
`uv` opportunistically links with `xz`, so I think we should stick with the normal port dependency declaration